invisible_cities/reco/corrections_test.py

```diff
@@ -1,13 +1,14 @@
 from collections   import namedtuple
 import numpy as np
 
-from ..core             import fit_functions as fitf
-from ..core.exceptions  import ParameterNotSet
-from ..reco.corrections import Correction
-from ..reco.corrections import Fcorrection
-from ..reco.corrections import LifetimeCorrection
-from ..reco.corrections import LifetimeRCorrection
-from ..reco.corrections import LifetimeXYCorrection
+from ..core                import fit_functions as fitf
+from ..core.exceptions     import ParameterNotSet
+from ..reco.corrections    import Correction
+from ..reco.corrections    import Fcorrection
+from ..reco.corrections    import LifetimeCorrection
+from ..reco.corrections    import LifetimeRCorrection
+from ..reco.corrections    import LifetimeXYCorrection
+from ..icaro.hst_functions import poisson_sigma
```

Comment:
Please move `poisson_sigma` to `core_functions.py`. The `icaro` package will (probably) be removed in the near future.